guides/source/upgrading_ruby_on_rails.md

```diff
@@ -884,8 +884,8 @@ end
 
 This can be turned off per-association with `optional: true`.
 
-This default will be automatically configured in new applications. If existing application
-want to add this feature it will need to be turned on in an initializer.
+This default will be automatically configured in new applications. If existing applications
```

Comment:
This sentence can either be:
> If an existing application wants to add this feature it will need to be turned on in an initializer.

Or:

> If existing applications want to add this feature they will need to be turned on in an initializer.

Lets add the first one.

Reply:
> If an existing application wants to add this feature it will need to be turned on in an initializer.
Plus one for this suggestion. 👍

Reply:
Changed to first suggestion.